crates/api/examples/hello.rs

```diff
@@ -35,7 +35,9 @@ fn main() -> Result<()> {
 
     // Compiler the `*.wasm` binary into an in-memory instance of a `Module`.
     println!("Compiling module...");
-    let module = HostRef::new(Module::new(&store, &binary).context("> Error compiling module!")?);
+    let module = HostRef::new(
+        Module::new(&store, &binary).map_err(|_| format_err!("> Error compiling module!"))?,
```

Comment:
How come this changed? I thought it was largely only instantiation that changed?

Reply:
I wanted to have it in one "style": context() just did not fit. I can return it back.